Api.Flickr/Models/FlickrPhotosetsGetPhotosResponseStatus.cs

```diff
@@ -0,0 +1,15 @@
+namespace Api.Flickr.Models
+{
+    /// <summary>
+    /// Statuses of response for photosets.getPhotos method
+    /// </summary>
+    public enum FlickrPhotosetsGetPhotosResponseStatus
+    {
+        Okay,
+        PhotosetNotFound,
```

Comment:
404

Reply:
Flickr API provides their custom codes for errors so it's redundand